client-common/Cargo.toml

```diff
@@ -23,6 +23,7 @@ jsonrpc = { version = "0.11", optional = true }
 serde_json = { version = "1.0", optional = true }
 parity-scale-codec = { features = ["derive"], version = "1.0" }
 websocket = { version = "0.24", default-features = false, features = ["sync"], optional = true }
+tendermint = { git = "https://github.com/yihuang/tendermint-rs.git", rev="95823039071a44dabc24825b777d3c5b95a6665b" }
```

Comment:
@yihuang we can maintain the fork here: https://github.com/crypto-com/tendermint-rs

but we should try to move to the upstream interchain one when possible, so feel free to open PRs on the upstream interchain one with changes in the fork

Reply:
OK, the biggest changes might be the adding of a bunch of ``Default`` instances.

Reply:
Also, there seems no way to use a feature to make the ring dependency conditional, maybe we have to upgrade the secure_connection to newer ring library before we can merge back to them ;(

Reply:
Just created the first PR to upgrade ``ring`` to ``0.16.9``. https://github.com/interchainio/tendermint-rs/pull/60